web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -153,6 +160,33 @@ export function SendForm(props: SendFormProps) {
     <Spinner size={12} customClass="inline-block" />
   );
 
+  useEffect(() => {
+    setError({ recipient: '' });
+
+    if (mnsExtension.test(recipient)) {
+      const inputMns = recipient.replace(mnsExtension, '');
+      resolveDns(inputMns);
+    } else {
+      setMnsAddressCorrelelation(false);
+      resetTargetMnsAddress();
+    }
+
+    if (targetMnsAddress && checkAddressFormat(targetMnsAddress)) {
+      setMnsAddressCorrelelation(true);
+    } else {
+      setMnsAddressCorrelelation(false);
+    }
+  }, [recipient, targetMnsAddress]);
+
+  function confirmMnsAddress(): void {
+    if (targetMnsAddress && mnsExtension.test(recipient)) {
+      setRecipient(targetMnsAddress);
+    } else {
+      setMnsAddressCorrelelation(false);
```

Comment:
I guess it's supposed to be `Correlation` ? Please make sure the issue isn't in other files too
```suggestion
      setMnsAddressCorrelation(false);
```